templates/terms-of-reference.md

```diff
@@ -0,0 +1,26 @@
+# Name of Group
+## Purpose/role of the group:
+- What is the broad purpose and objectives
+- when was the group established and by whom
+- what are the aims and responsibilities of the group
+- how this is differing from other groups.
+## Membership:
+- Who's in it, who's it open to, are there restrictions on numbers, are there reps from all groups, how long does someone need to be on the group, what are their obligations (eg do they have to turn up to the conferences for example?)
+- does the membership makeup include current conference organisers for all conferences etc
+## Accountability
+- how is this group going to report back on the activities of the group,
+- what is it's accountabilities to the stakeholders (conference organisors, attendees, council, etc)
+## Review
+- how often will the group review the relevance and value of it's work and this terms of reference?
+## Working methods/ways of working
+- eg face to face? online? are there sub groups?
+## Meetings
+- how many will there be
+- who will organise and chair
+- what type of things are on the agenda
+- how will things be escalated
+- can non-members turn up?
+- are secretarial duties required?
+## Definition of terms
+- provide defitions of any key terms/acronyms etc
+
```

Comment:
One question NOT here is "Do you already have a constitution/written rules (and how would we view such a thing)?"